protocol.md

```diff
@@ -870,6 +902,12 @@ export interface TextDocumentClientCapabilities {
 ```typescript
 interface ClientCapabilities {
 	/**
+	 * The client supports receiving the result solely through $/partialResult notifications for requests from the client to the server.
+	 * If true, the server should send all data through $/partialResult and send null as the result in the final response.
```

Comment:
Okay, got it. This likely does not affect single threaded runtimes like Javascript?
As I said though, using our own IDs etc. does not solve this. JSON Patches need to be applied in the right order in any case (swapping opertations results in a different document), and the JSON RPC library should make sure messages are emitted in the order they come in.